.github/workflows/docker-publish.yml

```diff
@@ -9,9 +9,17 @@ on:
 jobs:
   docker:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      security-events: write
```

Comment:
why we need this permissions?

Reply:
this is to setup another PR after this one, allowing automated security scans of the images ( like this: https://github.com/KingPin/php-docker/security/code-scanning ) if you wish I can remove it here and wait till the second PR.

Reply:
you should delete it because its no reason in this PR

A PR should only focus on one thing